javascript/javascript3/week3/circleDraw/main.js

```diff
@@ -0,0 +1,107 @@
+class Circle {
+    constructor(x, y, r, startAngle, endAngle, fillColor) {
+        this.x = x;
+        this.y = y;
+        this.r = r;
+        this.startAngel = startAngle;
+        this.endAngel = endAngle;
+        this.fillColor = fillColor;
+    }
```

Comment:
Your circles' start and end angles are always the same, so you don't need to pass them in the constructor. You can just declare the variable value instead of passing it every time!

Reply:
Hi, I have used constructor as per assignment. constructor(x, y, r, startAngle, endAngle, fillColor)